package.json

```
@@ -13,6 +13,7 @@
     "h1z1-server-demo-2016": "scripts/h1z1-server-demo-2016.js"
   },
   "dependencies": {
+    "@types/js-yaml": "^4.0.5",
```

```suggestion
    "@types/js-yaml": "4.0.5",
```
Use fixed version ( so it need to redo the package-lock )